048-IncrementalSynapsePipelines/Student/Challenge-00.md

```diff
@@ -0,0 +1,46 @@
+# Challenge 0 - Setup the source and target environments
+
+**[Home](../README.md)** - [Next Challenge>](Challenge-01.md)
+
+## Introduction
+
+We need to setup the proper environment for the Hackathon.  Thus, we need everyone on the team to have access to the Azure SQL and Synapse Environments.  Also, any ancilliaries resources such as the Azure Storage Account and Key Vault.  
+
+## Description
```

Comment:
should you provide ARM templates as a starting point?

Reply:
I agree in a real environment with Dev/Test/Production that this should be ARM templated but for this exercise I would rather them talk it through and understand how to build this out manually and discuss all the items needed and how to configure them before getting into ARM templates.  Hope that makes sense.